ZException.cs

```diff
@@ -63,15 +63,15 @@ protected ZException()
 		/// Initializes a new instance of the <see cref="ZException"/> class.
 		/// </summary>
 		/// <param name="errorCode">The error code returned by the ZeroMQ library call.</param>
-		internal ZException(ZError errorSymbol)
+		public ZException(ZError errorSymbol)
 			: this(errorSymbol, default(string), default(Exception))
 		{ }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ZException"/> class.
         /// </summary>
         /// <param name="errorCode">The error code returned by the ZeroMQ library call.</param>
-        internal ZException(ZError errorSymbol, string message)
+        public ZException(ZError errorSymbol, string message)
```

Comment:
Why do these need to be public? Why should someone from outside instantiate this exception type?
An explanation should be added to the code, such that it is not changed accidentially again, since it is not obvious.

Reply:
they MUST be public, because you sometimes want to do

	if (error == ZError.ETERM)
		return;	// Interrupted
	throw new ZException(error);

Reply:
You would want to throw a clrzmq4 exception *outside* of clrzmq4? This appears very strange to me.

Reply:
No, this isn't. You also don't complain to throw a FileNotFoundException, just because it is stated in the System library.